app/controllers/plans_controller.rb

```diff
@@ -36,36 +44,47 @@ def create
     @plan = Plan.new
     authorize @plan
 
-    # We set these ids to -1 on the page to trick ariatiseForm into allowing the autocomplete to be blank if
-    # the no org/funder checkboxes are checked off
-    org_id = (plan_params[:org_id] == '-1' ? '' : plan_params[:org_id])
-    funder_id = (plan_params[:funder_id] == '-1' ? '' : plan_params[:funder_id])
+    # We set these ids to -1 on the page to trick ariatiseForm into allowing the
+    # autocomplete to be blank if the no org/funder checkboxes are checked off
+    org_id = (plan_params[:org_id] == "-1" ? "" : plan_params[:org_id])
+    funder_id = (plan_params[:funder_id] == "-1" ? "" : plan_params[:funder_id])
 
-    # If the template_id is blank then we need to look up the available templates and return JSON
+    # If the template_id is blank then we need to look up the available templates and
+    # return JSON
     if plan_params[:template_id].blank?
       # Something went wrong there should always be a template id
       respond_to do |format|
-        flash[:alert] = _('Unable to identify a suitable template for your plan.')
+        flash[:alert] = _("Unable to identify a suitable template for your plan.")
         format.html { redirect_to new_plan_path }
       end
     else
       # Otherwise create the plan
-      @plan.principal_investigator = current_user.surname.blank? ? nil : "#{current_user.firstname} #{current_user.surname}"
+      @plan.principal_investigator = if current_user.surname.blank?
+                                       nil
+                                     else
+                                       "#{current_user.firstname} #{current_user.surname}"
+                                     end
       @plan.principal_investigator_email = current_user.email
 
-      orcid = current_user.identifier_for(IdentifierScheme.find_by(name: 'orcid'))
+      orcid = current_user.identifier_for(IdentifierScheme.find_by(name: "orcid"))
       @plan.principal_investigator_identifier = orcid.identifier unless orcid.nil?
 
       @plan.funder_name = plan_params[:funder_name]
 
-      @plan.visibility = (plan_params['visibility'].blank? ? Rails.application.config.default_plan_visibility :
-                                                             plan_params[:visibility])
+      @plan.visibility = if plan_params["visibility"].blank?
+                           Rails.application.config.default_plan_visibility
+                         else
+                           plan_params[:visibility]
+                         end
 
       @plan.template = Template.find(plan_params[:template_id])
 
       if plan_params[:title].blank?
-        @plan.title = current_user.firstname.blank? ? _('My Plan') + '(' + @plan.template.title + ')' :
-                                    current_user.firstname + "'s" + _(" Plan")
+        @plan.title = if current_user.firstname.blank?
+                        _("My Plan") + "(" + @plan.template.title + ")"
```

Comment:
Most style guides in Ruby prefer double quotes over single quotes, because double quotes provide more functionality (like interpolation, escaping characters, etc.)

There's a slight performance difference between single and double quotes, in favour of single quotes, but that becomes essentially irrelevant when we're using `# frozen_string_literal: true`

Reply:
👍 That works for me.
I just ran rubocop on a controller I am working on and got the "prefer double quotes" message.